content/2015.12.loadtesting.with.oauth.rst

```diff
@@ -0,0 +1,171 @@
+How to load test a OAuth-protected HTTP API?
+############################################
+
+:lang: en
+:date: 2015-12-02
+:url: load-testing-a-http-api-which-uses-oauth-for-authentication
+:summary: This article explains how to test the performance of an HTTP API when its access depends on an external OAuth service.
+
+
+What is OAuth and how do you use it to authenticate an HTTP API?
+================================================================
+
+`OAuth <https://en.wikipedia.org/wiki/OAuth>`_ is a protocol that let
```

Comment:
s/let/lets/, but you might also consider re-wording to something like "allows a user to delete"